pkg/node/store/store.go

```diff
@@ -32,6 +36,14 @@ var (
 	// break backwards compatibility
 	NodeStorePrefix = path.Join(kvstore.BaseKeyPrefix, "state", "nodes", "v1")
 
+	// NodeRegisterStorePrefix is the kvstore prefix of the shared
+	// store for node registration that will be followed by k8s
+	// namespace name depending on agent's --k8s-namespace option
```

Comment:
Even in the comment you refer to the other flag :smile: it should be `join-k8s-namespace`

Reply:
historical reasons, will fix.

Reply:
The fix was to remove the latter part of the comment. Namespace is not part of the key path, it is (only) in the node labels.